R/count_watershed_data.R

```diff
@@ -221,9 +221,30 @@ count_watershed_data <- function(data_dir,
 
         #-------------------------------------------------------
         # TELECONNECTION - WASTE WATER TREATMENT PLANTS
-        wasteflow_points[watersheds_select, ] -> wasteflow_select
+        wasteflow_points[watersheds_select, ] %>%
+          as.data.frame() %>%
+          dplyr::select(c("FACILITY_NAME",
+                          "CWNS_NUMBER",
+                          "PRIMARY_WATERSHED_HUC",
+                          "PRIMARY_WATERSHED_NAME",
+                          "lon",
+                          "lat",
+                          "DISCHARGE_METHOD",
+                          "EXIST_TOTAL",
+                          "PROJ_TOTAL",
+                          "PRES_RES_RECEIVING_COLLCTN",
+                          "PRES_RES_ONSITE_WTS"
+          )) -> wasteflow_select
 
+        wasteflow_select[is.na(wasteflow_select)] <- 0
```

Comment:
best to keep `NA` as `NA`. Instead just add the argument `na.rm = T` to the sums below.